GameData/Halloween/Localization/no-no.cfg

```diff
@@ -0,0 +1,86 @@
+// no-no.cfg v1.1.0.0
+// Halloween (HLWN)
+// created: 05 Oct 21
+// updated: 20 Oct 2022
+
+// translated by: [@tplive](https://github.com/tplive)
+// CC BY-SA 4.0 by  zer0Kerbal
+
+Localization
+{
+	en-us
+	{
+	// ************** General
+		#HLWN-Candle-manu = Lunatic Aeronautics
+		#HLWN-kraken-manu = Dr Hoeydonck Van Kerman Kermnassus
+		#HLWN-strutCon-manu = Ungoliant Edderkopp Emporium
+
+		#HLWN-tags = hlwn (jack )lantern pumpkin halloween brown charlie eve porkjet nohark nil2work
+
+	// ************** Halloween Candle
+		#HLWN-Candle-titl-s = Halloween-stearinlys (Lite)
+		#HLWN-Candle-titl-m = Halloween-stearinlys
+		#HLWN-Candle-titl-l = Halloween-stearinlys (Stor)
```

Comment:
Stort?

```suggestion
		#HLWN-Candle-titl-l = Halloween-stearinlys (Stort)
```

Reply:
"Lys" is both singular and plural. Because the word is neuter, "lite", "vanlig", and "stort" would be the correct translations for "small", "regular", and "large". Whether "regular"/"vanlig" is necessary is up to you.

Reply:
Also "lys" (here) means any kind of light source, rather than being just a "candle". So while "lys" is more terse, "stearinlys" is specific. If "stearinlys" doesn't fit the UI, or it's otherwise obvious that this is indeed a candle, then "lys" would be fine.

Reply:
thank you both @tplive and @boyum !

It is so wonderful to see this conversation.